dashboard/src/reducers/config.ts

```diff
@@ -0,0 +1,20 @@
+import { getType } from "typesafe-actions";
+
+import actions from "../actions";
+import { ConfigAction } from "../actions/config";
+import { IConfig } from "../shared/Config";
+
+const initialState: IConfig = {
+  namespace: "default",
```

Comment:
do we really want a default? I am wondering if this could have side-effects in the case that there are problems while retrieving the value from the config endpoint.

Reply:
It's possible that there's a race condition here, yes. I need to look into this more.

Reply:
fixed, PTAL @migmartri